runtime/shiden/src/lib.rs

```diff
@@ -623,7 +618,6 @@ impl pallet_vesting::Config for Runtime {
 parameter_types! {
     pub const DepositPerItem: Balance = MILLISDN / 1_000_000;
     pub const DepositPerByte: Balance = MILLISDN / 1_000_000;
-    pub const MaxValueSize: u32 = 16 * 1024;
```

Comment:
Can you explain why you remove this const ?

Reply:
Dead code.